### DISCUSSION TITLE
[Frage] Ähnliche Farben auswählen

### DISCUSSION THREAD

**Discussion by Blackcat13140** [tannerhelland/PhotoDemon#583] [Q&A]
Ich hatte früher Photoshop und bin auf der Suche nach einer Alternativen. Dabei bin ich auf PhotoDemon gestoßen und habe dazu eine Frage. Beim Photoshop gab es die Möglichkeit eine Auswahl nach ähnlichen Farben durchzuführen und anschließend zu löschen. Mit dem Zauberstab kann man nur zusammenhängende Farben auswählen. Wenn aber im Bild viele vereinzelte Farbpunkte  hat, ist es damit sehr aufwendig. Gibt es die Funktion nach Farben im PhotoDemon auszuwählen?

**Comment by hi5** [comment]
You can click on the `Tolerance` options and select `continuous` or `global` and use the other options to adjust the selected color-range and regions:

![capture](https://github.com/user-attachments/assets/de1a26d0-ff28-40f4-af60-8291ac3543ac)

**Reply by Blackcat13140** [reply]
Thank you very much.
